README.md

```diff
@@ -27,6 +27,24 @@ Include the library in your sketch
 #include <ConfigManager.h>
 ```
 
+Create your `config` and `meta` structs. These are the definitions for your settings.
+
+```cpp
+struct Config {
+    char name[20];
+    bool enabled;
+    int8_t hour;
+    char password[20];
+} config;
+
+struct Metadata {
+    int8_t version;
+} meta;
+
+```
+
+> Note: These are examples of possible settings. The `config` is required, these values are arbitrary.
```

Comment:
`password` `hour` `enabled` tripped me up in regards to usage versus arbitrary custom settings